fix: Update Custom Component Maker

edwinjosechittilappilly opened the pull request:
Change error default system message value in the Anthropic component.

ogabrielluiz reviewed (dismissed):
Hey @edwinjosechittilappilly 

Can you check the ChatInput value on this template?

edwinjosechittilappilly commented:
@ogabrielluiz  I have removed  the incorrect inputs and replaced them with blank components. Additionally, I updated the output components with the latest versions.

carlosrcoelho reviewed (approved):
LGTM

edwinjosechittilappilly commented:
@ogabrielluiz  can you approve this PR?

edwinjosechittilappilly commented:
Requesting review from @Cristhianzl

Cristhianzl reviewed (approved):
lgtm
